### PR TITLE
Feature File for AV-86

### PR DESCRIPTION
**Pull request by Abhishek8694**
Hi Neha,
pls check the feature file for the story AV-86,
and reply if any .

Thanks,
Abhishek Vaidya

### CODE REVIEW

File: src/test/resources/features/productDescription.feature [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    Given I want to write a step with precondition
+    And some other precondition
+    When I complete action
+    And some other action
+    And yet another action
+    Then I validate the outcomes
+    And check more outcomes
+    
+     @tag2
+  Scenario: Title of your $cen@r!o
```
Comment (nehadubey45) [inline src/test/resources/features/productDescription.feature]: Please check the spelling of scenario. plz correct it

File: src/test/resources/features/productDescription.feature [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    Given I want to write a step with precondition
+    And some other precondition
+    When I complete action
+    And some other action
+    And yet another action
+    Then I validate the outcomes
+    And check more outcomes
+    
+     @tag3*9/2
```
Comment (nehadubey45) [inline src/test/resources/features/productDescription.feature]: Please remove char and symbols '*9/2' from @tag3

**Review by nehadubey45 — COMMENTED** [review]
Please make the changes as per suggestion

File: src/test/resources/features/productDescription.feature (hunk 1 above)
Comment (Abhishek8694) [reply src/test/resources/features/productDescription.feature]: made changes

File: src/test/resources/features/productDescription.feature (hunk 2 above)
Comment (Abhishek8694) [reply src/test/resources/features/productDescription.feature]: made changes

**Comment by nehadubey45** [comment]
Alright Changes are taken in consideration. now you can merge the feature file into master branch

**Pull request merged**